app/models/wiki_content/revision.rb

```diff
@@ -68,16 +68,18 @@ def plagiarism_report_link
     "/recent-activity/plagiarism/report?ithenticate_id=#{ithenticate_id}"
   end
 
-  WIKITEXT_REF_TAGS = 'feature.wikitext.revision.ref_tags'
-  WIKIDATA_REFERENCES = 'feature.len(<datasource.wikidatawiki.revision.references>)'
+  WIKITEXT_REF_TAGS = 'feature.wikitext.revision.ref_tags'
+  WIKIDATA_REFERENCES = 'feature.len(<datasource.wikidatawiki.revision.references>)'
+  WIKITEXT_SHORT_TEMPLATE = 'feature.enwiki.revision.shortened_footnote_templates'
+  WIKITEXT_REF_TAGS_SHORT_TEMPLATE = WIKITEXT_REF_TAGS + WIKITEXT_SHORT_TEMPLATE
 
   def ref_tags
-    features[WIKITEXT_REF_TAGS] || features[WIKIDATA_REFERENCES]
+    features[WIKITEXT_REF_TAGS_SHORT_TEMPLATE] || features[WIKIDATA_REFERENCES]
   end
 
   def ref_tags_previous
     return 0 if new_article
-    features_previous[WIKITEXT_REF_TAGS] || features_previous[WIKIDATA_REFERENCES]
+    features_previous[WIKITEXT_REF_TAGS_SHORT_TEMPLATE] || features_previous[WIKIDATA_REFERENCES]
```

Comment:
The same thing here

Reply:
You can run tests on your terminal to check errors before pushing the PR,
you can install rubocop and call `bundle exec rubocop`  to check for any warning or Offenses

Run `bundle exec` to run all the tests